src/libraries/System.Runtime/tests/System/Attributes.cs

```diff
@@ -533,6 +533,35 @@ public static void NegTest8()
             AssertExtensions.Throws<ArgumentException>(null, () => (ArgumentUsageAttribute)Attribute.GetCustomAttribute(paramInfos[0], attributeType, false));
 
         }
+
+        // reproduces https://github.com/dotnet/runtime/issues/64335
+        [Fact]
+        [SkipOnMono("Mono throws on open generic type passed to GetCustomAttributes")]
```

Comment:
I beleive we should fix it in mono too, there is a [CoreLib project](https://github.com/dotnet/runtime/tree/main/src/mono/System.Private.CoreLib) within mono source

Reply:
Looks like [generic attributes are not supported](https://github.com/dotnet/runtime/issues/56887) on mono at all, so let keep skipping mono test.